docs/getting-started.md

```diff
@@ -185,17 +185,37 @@ After each change, you can continue updating your component by using: `odo push
 
 ### 6. Adding storage to the component
 
-Now that you've got your component running, how do you add persistent any data between restarts?
+Now that you've got your component running, how do you persist data between restarts?
 
 If you wish to add storage to your component, `odo` makes it very easy for you to do this:
 
 ```console
 $ odo storage create nodestorage --path=/opt/app-root/src/storage/ --size=1Gi 
- OK  Added storage nodestorage to nodejs-ex-nodejs-nnjf
+ ✓   Added storage nodestorage to nodejs-ex-nodejs-nnjf
 ```
 
 That's it! Storage has been added your component with an allocated size of 1 Gb.
 
+## Command completion
+
+**Command completion is currently only supported for `bash`, `zsh` and `fish` shells.**
+
+`odo` provides smart completion of command parameters based on user input. For this to work, `odo` needs to integrate with the 
+executing shell. This is accomplished by running `odo --complete` and pressing `y` when asked to install the completion hook. 
```

Comment:
I don't like how we have the instructions for installation in the middle of a sentence. This isn't the same as the other instructions in this guide.

Could we space it out similar to my example?

This can be installed automatically by running:

```sh
odo --complete
```

This modifies your local shell configuration file, for example: `.bashrc` for bash.

You can modify your shell configuration manually by adding Odo:

```
complete -o nospace -C <full path to your odo binary> odo
```